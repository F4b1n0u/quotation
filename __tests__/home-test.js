import React from 'react'
import Home from '#components/home'
import renderer from 'react-test-renderer'

import { ThemeProvider } from 'styled-components/native'

const theme = {
  background: '#EAEAEA',
  box: '#FFFFFF',
  highlight: '#19A5B0',
}

test('renders correctly', () => {
  const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <Home
        onGoToInsight={() => {}}
        onGoToCheckIn={() => {}}
      />
    </ThemeProvider>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})