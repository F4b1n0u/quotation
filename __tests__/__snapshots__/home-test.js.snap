// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#EAEAEA",
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "space-around",
      },
    ]
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 10,
        "height": "30%",
        "justifyContent": "center",
        "opacity": 1,
        "width": "70%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#19A5B0",
            "fontSize": 50,
          },
        ]
      }
    >
      Check In
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 10,
        "height": "30%",
        "justifyContent": "center",
        "opacity": 1,
        "width": "70%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#19A5B0",
            "fontSize": 50,
          },
        ]
      }
    >
      Insight
    </Text>
  </View>
</RCTSafeAreaView>
`;
